backend/.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true,
+    // jest: true,
+  },
+  parser: '@typescript-eslint/parser',
+  plugins: ['prettier', '@typescript-eslint' /*, 'jest' */],
+  extends: [
+    'standard',
+    'eslint:recommended',
+    'plugin:prettier/recommended',
+    'plugin:@typescript-eslint/recommended',
+  ],
+  // add your custom rules here
+  rules: {
+    'no-console': ['error'],
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
```

Comment:
I suggest always error.

Reply:
Ok, why do we have this in the frontend? Vue? Debugger?

Reply:
fixed it tho <3